---
apiVersion: "v1"
kind: "Event"
metadata:
  labels:
    EntandoResourceKind: "EntandoApp"
    EntandoResourceNamespace: "my-namespace"
    EntandoApp: "my-app"
  name: "my-app-77701884"
  namespace: "my-namespace"
  ownerReferences:
  - apiVersion: "entando.org/v1"
    kind: "EntandoApp"
    blockOwnerDeletion: true
    controller: true
    name: "my-app"
    uid: "cc0ad632-ab4f-404f-afc7-b2f56b0bcf45"
action: "FAILED"
count: 1
firstTimestamp: "2021-06-28T12:41:14Z"
involvedObject:
  apiVersion: "entando.org/v1"
  kind: "EntandoApp"
  fieldPath: "status"
  name: "my-app"
  namespace: "my-namespace"
  resourceVersion: "7968018"
  uid: "cc0ad632-ab4f-404f-afc7-b2f56b0bcf45"
lastTimestamp: "2021-06-28T12:41:14Z"
message: "The deployment of EntandoApp my-namespace/my-app failed due to Test Exception.\
  \ Fix the root cause and then trigger a redeployment by adding the annotation 'entando.org/processing-instruction:\
  \ force'"
reason: "Failed"
related:
  apiVersion: "v1"
  kind: "Pod"
  name: "entando-k8s-entando-k8s-app-con-jhlxz-24-from-build-pack--zzpbs"
  namespace: "controller-namespace"
source:
  component: "entando-k8s-app-controller"
type: "Error"
